articles/libraries/machine_learning/concepts/Qccc.Introduction.md

```diff
@@ -0,0 +1,33 @@
+
+# Framework and goals
+Quantum encoding and processing of information is a powerful alternative to traditional predictive data analysis (that uses tools such as support vector machines, neural nets, decision trees etc.) Quantum classifiers, in particular, encode data in quantum registers that are concise relative to the number of features, systematically employ quantum entanglement as computational resource and employ quantum measurement for class inference. Circuit centric quantum classifier is a relatively simple quantum solution that combines data encoding with a rapidly entangling/disentangling quantum circuit followed by measurement to infer class labels of data samples. The goal is to ensure classical characterization and storage of subject circuits, as well as hybrid quantum/classical training of the circuit parameters even for extremely large feature spaces.
+
+# Classifier architecture
+Classification is a supervised machine learning task, where the goal is to infer class labels $\{y_1,y_2,\ldots,y_d\}$ of certain data samples. The `training data set` is a collection of samples $\mathcal{D}=\{(x,y)}$ with known pre-assigned labels. Here $x$ is a data sample and $y$ is its known label called `training label`.
+Somewhat similar to traditional methods, quantum classification consists of three steps: data encoding, preparing of a classifier state, taking a measurement. Due to probabilistic nature of the measurement, these three steps must be repeated multiple times. The measurement may be viewed as a quantum equivalent of some non-linear activation.
+Both the encoding and the computing of the classifier state are done by means of `quantum circuits`. While the encoding circuit is usually data-driven and parameter-free, the classifier circuit contains a sufficient set of learnable parameters. In the proposed solution the classifier circuit is composed of single-qubit rotations and two-qubit controlled rotations. The learnable parameters here are the rotation angles. The rotation and controlled rotation gates are known to be `universal` for quantum computation, which means that any unitary weight matrix can be decomposed into a long enough circuit consisting of such gates. Our architecture explores relatively shallow circuits, which therefore must be `rapidly entangling` in order to capture all the correlations between the data features at all ranges. An example of the most useful rapidly entangling circuit component is shown on FIG 1. Even though this circuit consists of only $3\,n+1$ gates, the unitary weight matrix that it computes ensures significant cross-talk between $2^n$ features.
+ 
+
+## FIG. 1
```

Comment:
We generally haven't used manuscript-style figure labeling in conceptual docs.
This in particular would format "FIG.1" as a section header rather than as a caption; if we decide on captions, it might be good to look back at the docs authoring guide for details as to the best way to do that.

Reply:
At the moment, I do not know how to handle this. Moving on down the list for now.

Reply:
I'd suggest looking to other examples in the documentation, for example the page at https://docs.microsoft.com/quantum/libraries/numerics/installation. Generally, we've handled figures as being entirely inline, without captions, as that renders well on a wide variety of mobile and desktop agents. For accessibility, I'd suggest providing alternative text in preference to a caption.

Reply:
I have strived to inline the figure. Also took heed of the section/subsection level warnings